### DISCUSSION TITLE
Navigate to a state using DevTools

### DISCUSSION THREAD

**Discussion by LucasBersier** [vuejs/devtools-v6#2034] [Q&A]
Is there a way to navigate to a specific state using the devtools? All of the states show up under the Routes dropdown, but I would like to navigate to them to debug issues with a specific state.

If not, is there a way to do this outside of devtools, such as in the console to generate transition events?

> https://github.com/vuejs/router/discussions/1728

**Comment by 680614** [comment]
> Is there a way to navigate to a specific state using the devtools? All of the states show up under the Routes dropdown, but I would like to navigate to them to debug issues with a specific state.
> 
> If not, is there a way to do this outside of devtools, such as in the console to generate transition events?
> 
# - [ ] @tsukkee ~~~~
